README.md

```diff
@@ -1,5 +1,9 @@
+<div class="hide-in-mkdocs">
+
 # Synthetic Data Values Generator (SDVG)
```

Comment:
I think that you should add a link for the new documentation in the readme files

Reply:
fixed